### PR TITLE
feat: roadmap

### LINKED ISSUE #6734: [FEATURE] Roadmap
### Description

A new page to show...

- recently released features
- features being worked on
- future planned features

GitKraken do it well https://www.gitkraken.com/git-client/roadmap

Screenshot 2023-05-10 at 18 57 58

Thank you @dan-mba for the reminder and nudge on this 💯 

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by eddiejaoude**
## Fixes Issue

closes #6734 

Screenshot 2023-05-10 at 18 56 05

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6735"></a>

### CODE REVIEW

File: pages/roadmap.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  CheckIcon,
+  ComputerDesktopIcon,
+  ClockIcon,
+} from "@heroicons/react/24/outline";
+
+import Page from "@components/Page";
+import PageHead from "@components/PageHead";
+import Link from "@components/Link";
+
+export default function Changelog() {
```
[the remark this answers is not in the document]
Comment (eddiejaoude) [reply pages/roadmap.js:11]: Oh good spot 👍 I think the change is only required here

[the remark this answers is not in the document]
Comment (Ananya2001-an) [reply pages/roadmap.js]: Nice 👍🏼

**Comment by ChinmayMhatre** [comment]
I have changed the card to have a border instead of a ring since it looked inconsistent and added the dark mode classes. 

Do let me know if colour needs to be changed.🙌🏻

**Comment by eddiejaoude** [comment]
Looks great! Thank you 👍

**Review by SaraJaoude — APPROVED** [review]
looks great!

**Pull request merged**